pages/common/identify.md

```diff
@@ -0,0 +1,7 @@
+# identify
+
+> Image magick command.
+
+- Collect dimensions of all jpeg files under current directory:
+
+`identify -format "%f,%w,%h\n" *.jpg > filelist.csv`
```

Comment:
```suggestion
`identify -format "%f,%w,%h\n" *.{{jpg}} > {{filelist.csv}}`
```

Reply:
`filelist.csv` → `path/to/filelist.csv`, @zdroid?